nodeCf.js

```diff
@@ -1,176 +1,207 @@
-var Promise = require('bluebird');
-var nunjucks = require("nunjucks");
-var _ = require('lodash');
-var Ajv = require('ajv');
-var fs = require('fs');
+const Promise = require('bluebird');
```

Comment:
I'm certainly not opposed to using a transpiler to get that functionality, but thus far haven't given it much thought.